source/local/local.go

```diff
@@ -41,6 +42,52 @@ type Source struct{}
 func (s Source) Name() string { return "local" }
 
 func (s Source) Discover() (source.Features, error) {
+	hooksFeatures, err := getFeaturesFromHooks()
+	if err != nil {
+		log.Printf(err.Error())
+	}
+
+	readsFeatures, err := getFeaturesFromReads()
+	if err != nil {
+		log.Printf(err.Error())
+	}
+
+	// Merge features from hooks and reads
+	for k, v := range hooksFeatures {
+		readsFeatures[k] = v
```

Comment:
Should you print an error or log if you endup overriding a value?
Before hooks this wasn't handled in any way, should this change?

Reply:
> Should you print an error or log if you endup overriding a value?
> Before hooks this wasn't handled in any way, should this change?

Good point, probably indicating this in log would be worthwhile. I wouldn't print an error as I can think of cases where overriding could be used.

There would be three point in the code where this should be done:
1. inside `getFeaturesFromHooks()`, checking for hook overriding features from other hook
2. inside `getFeaturesFromReads()`, checking for file overriding features from other file
3. inside `Discover()`, checking for hooks overriding features from files

Reply:
Done